releasenotes/notes/ci-vis-gzip-payload-a93c4b5d4687f1c4.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 features:
   - |
-    CI Visibility: This introduces the ability to gzip the payload when using the agentless setup, incurring in less network bandwith consumption.
+    CI Visibility: This introduces the ability to gzip the payload when using the agentless setup, incurring in less network bandwidth consumption.
```

Comment:
nit: this typo still exists in the changelog https://github.com/DataDog/dd-trace-py/blob/main/CHANGELOG.md#380